watson/config.py

```diff
@@ -60,6 +62,42 @@ def getboolean(self, section, option, default=False):
         val = self.get(section, option)
         return val.lower() in ('1', 'on', 'true', 'yes') if val else default
 
+    def getlist(self, section, option, default=None):
+        """
+        Return value of option in given section as a list of strings.
+
+        If option is not set, return default (defaults to an empty list).
+
+        The option value is split into list tokens using one of two strategies:
+
+        * If the value contains any newlines, i.e. it was written in the
+          configuration file using continuation lines, the value is split at
+          newlines and empty items are discarded.
```

Comment:
The test already covers that with `value1` in the test config, where the first option line is empty. But I will add another empty line on its own, to demonstrate that this works too.